Modules/RPC.cs

```diff
@@ -403,25 +409,20 @@ public static void SendExecutionerTarget(byte executionerId, byte targetId)
         }
         public static void CustomWinTrigger(byte winnerID)
         {
-            List<PlayerControl> Impostors = new();
+            if (!AmongUsClient.Instance.AmHost) return;
+
+            PlayerControl Winner = null;
```

Comment:
Not require

Reply:
Remove unused variable Winner and associated if statement?

Reply:
If statement require